test/extended/util/db/mysql.go

```diff
@@ -80,6 +81,11 @@ func (m MySQL) QueryPrivileged(oc *util.CLI, query string) (string, error) {
 			query, masterConf.Env["MYSQL_DATABASE"])).Output()
 }
 
+// QueryPrimary queries the database on primary node as a regular user.
+func (m MySQL) QueryPrimary(oc *util.CLI, query string) (string, error) {
+	return "", errors.New("not implemented")
```

Comment:
It's just matter of the naming, because, yes, in MySQL/PostgreSQL it will be master. If I rename it to the `QueryMaster()` than it would be less applicable to the MongoDB replicat set configuration... or it wouldn't?